src/packages/migrate/src/utils/seed.ts

```diff
@@ -0,0 +1,72 @@
+import fs from 'fs'
+import path from 'path'
+import execa from 'execa'
+
+export function detectSeedFiles() {
+  const seedPath = path.join(process.cwd(), 'prisma', 'seed.')
+
+  const jsFilePath = seedPath + 'js'
+  const tsFilePath = seedPath + 'ts'
+  const shFilePath = seedPath + 'sh'
+  const goFilePath = seedPath + 'go'
+
+  const detected = {
+    seedPath,
+    numberOfSeedFiles: 0,
+    js: fs.existsSync(jsFilePath) ? jsFilePath : null,
+    ts: fs.existsSync(tsFilePath) ? tsFilePath : null,
+    sh: fs.existsSync(shFilePath) ? shFilePath : null,
+    go: fs.existsSync(goFilePath) ? goFilePath : null,
+  }
+
+  detected.numberOfSeedFiles =
+    Number(!!detected.js) +
+    Number(!!detected.ts) +
+    Number(!!detected.sh) +
+    Number(!!detected.go)
+
+  return detected
+}
+
+export async function tryToRunSeed() {
+  const detected = detectSeedFiles()
+
+  if (detected.numberOfSeedFiles === 0) {
+    throw new Error(`No seed file found.
+Create a \`seed.ts\` or \`.js\` or \`.sh\` or \`.go\` file in the prisma folder.`)
+  } else if (detected.numberOfSeedFiles > 1) {
+    throw new Error(
+      `More than 1 seed file was found in \`${path.relative(
+        process.cwd(),
+        path.dirname(detected.seedPath),
+      )}\`.
+This command only supports 1 \`seed.ts\` or \`.js\` or \`.sh\` or \`.go\` file.`,
+    )
+  } else {
+    if (detected.js) {
```

Comment:
use switch case? :D

Reply:
Personally the switch cases don't bring so much value 🤷‍♂️

Reply:
I think it's more what it says than what it does; usually I use if-else if there are many different conditions which may be very different on what they do (like either throw or do some action, or create a database record or update something, etc.), but in this case it's the same action but a slightly different execution. It's more than a nit pick though so feel free to ignore me haha

Reply:
Thanks! Indeed I get the idea :)

Reply:
`switch` is for noobs, you should use a map! 😄